### PR TITLE
Made the testimonial section a slider

### LINKED ISSUE #89: [feat]: Make the testimonials section a slider
The current testimonials section is cool but it'd be nice to make it into a slider that auto-updates according to the number of slides.

Each slide will represent a testimonial.
Contributors can then add their own testimonial in the codebase.

### PR DESCRIPTION
**Pull request by publikphigor**
Changed the testimonial section to a slider that is dynamically generated based on the array of testimonials in the testimonials.js files.

Closes #89 

### Before
Screenshot (136)

### After
Screenshot (135)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,9 +102,27 @@ <h2>Helpful Links </h2>
...
+  
+  <div class="slider-controls">
+    <i class="fa-solid fa-circle-arrow-left slider-control slider--prev"></i>
+    <i class="fa-solid fa-circle-arrow-right slider-control slider--next"></i>
+    <!-- <ion-icon name="arrow-dropleft" class="slider-control slider--prev"></ion-icon>
+    <ion-icon name="arrow-dropright" class="slider-control slider--next"></ion-icon> -->
+  </div>
+  
+  <div class="slider">
+     <!-- THIS IS JUST A PLACEHOLDER ===> Edit the testimonial.js file to add a new testimonial. Create a new object in the array of testimonials with the format. -->
```
Comment (michaella23) [inline index.html:117]: Appreciate the code comment here!

File: testimonials.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/**
+ *          ADD A NEW OBJECT TO THE testimonials ARRAY BELOW TO CREATE A NEW TESTIMONIAL.
+ *          DON'T TOUCH THE REST OF THE CODE!!! (joking).
```
Comment (michaella23) [inline testimonials.js:3]: 😂

File: testimonials.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+];
+
+/**=================== SELECTORS ===================  */
+const sliderContainer = document.querySelector(".slider");
+const btnLeft = document.querySelector(".slider--prev");
+const btnRight = document.querySelector(".slider--next");
+let counter = 1;
+let slideSize, slides;
+
+/**=================== FUNCTIONS ===================  */
```
Comment (michaella23) [inline testimonials.js:41]: Your code looks really great. 👏

Comment (publikphigor) [reply testimonials.js:41]: Thank you 🚀🚀🚀

File: testimonials.js (hunk 2 above)
Comment (publikphigor) [reply testimonials.js:3]: Took time to write 😑🤣🤣

File: index.html (hunk 1 above)
Comment (publikphigor) [reply index.html:117]: I think it was necessary. Thank you

**Comment by publikphigor** [comment]
@michaella23 of course there are room for improvement. I can make them more visible.

I could also make the slider "play by itself" (a line of code will do this).

**Comment by michaella23** [comment]
@publikphigor sure! That works. It's up to you. And no rush!

File: index.html (hunk 1 above)
Comment (michaella23) [reply index.html:117]: Yes, if other people want to add a testimonial this will give them some direction. 🙂

**Comment by publikphigor** [comment]
Thank you. I'd definitely like to take some time.
I'd also love if other contributors are encouraged to add a testimonial.

They're free to break the code! 
It's why we're here in the first place 😂

**Comment by michaella23** [comment]
I hope other contributors will add a testimonial as well!

**Pull request merged**